pkg/controller/volume/attachdetach/util/util.go

```diff
@@ -65,7 +65,8 @@ func CreateVolumeSpec(podVolume v1.Volume, podNamespace string, pvcLister coreli
 				pvcSource.ClaimName,
 				err)
 		}
-
+		// Populating PodNamespace field of Volume.Spec with the namespace value passed as an argument to CreateVolumeSpec
+		volumeSpec.PodNamespace = podNamespace
```

Comment:
@sbezverk thank you for adding this.  Can you throw in some tests to make sure value is making it through.